docs/index.rst

```diff
@@ -56,8 +56,9 @@ Step 2: Setup Bioconda
 
 After installing Miniconda, you can use the ``conda`` command to setup bioconda with::
 
-   conda config --add channels r
    conda config --add channels conda-forge
```

Comment:
When I was trying it, `--add` will add the channel as the highest priority even if it already exists in `.condarc`. So if you do the `--add` commands in this order, the same set of commands can be used no matter what the existing state of .condarc.